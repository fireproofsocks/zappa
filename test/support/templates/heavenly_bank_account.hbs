He's got twenty million dollars
In his Heavenly Bank Account...
All from those chumps who was
{{dangerous_organization}}