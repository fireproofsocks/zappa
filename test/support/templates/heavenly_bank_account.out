He's got twenty million dollars
In his Heavenly Bank Account...
All from those chumps who was
&lt;script&gt;const born_again = kill(&quot;The unbelievers!); &quot;&lt;/script&gt;